src/events/messageCreate.js

```diff
@@ -4,7 +4,31 @@ const pointFunctions = require("../fileFunctions/pointFunctions.js");
 
 module.exports = async (bot, message) => {
 
-    if (message.channel.id == process.env.CHAN_ID_GAME && txtFunctions.isFileNotEmpty("./dataFiles/wordtofind.txt")) {
+    // Vérifier si l'auteur du message est l'auteur du mot à trouver
+
+    var idAuthor = "";
+
+    const channelId =  process.env.CHAN_ID_NEW_WORD;
+    const channelEvent = message.guild.channels.cache.get(channelId);
+
+    channelEvent.messages.fetch({ limit: 1 }).then(messages => {
+        const lastMessage = messages.first();
+
+        if (lastMessage.embeds.length > 0) {
+            const lastEmbed = lastMessage.embeds[0];
+
+            const mentionRegex = /<@(\d+)>/g;

```

Comment:
We should keep the autor when the new word is created. We have the information in parameters. We should store it in the wordtofind.txt next to the word to find with a delimitor.